Bottom: d8fb46f248ed2fa05c5f19929059a8829e6b197f
Top:    04a62ca6ff4c03f4f2c51b90a4ddbb7138166d45
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 0804c27..9451604 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1267,6 +1267,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
